homeworks/Naumkin/HW4/Index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link rel="stylesheet" href="media.css" type="text/css" />
+    <script
+      src="https://kit.fontawesome.com/4882f08fef.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+</html>
+<body>
+  <div id="account">
+    <header>varlamov.ru</header>
+    <div id="Name">
+      <span
+        ><img
+          id="foto"
+          src="https://i.pinimg.com/originals/69/45/2e/69452efaef6d6eaf75c564babaa294d8.png"
+          alt="My Photo"
+      /></span>
+      <span>
+        <h1>Make Russia warm again</h1>
+        <h3>Make Russia warm again</h3>
+        <h4>
+          <i class="fa-solid fa-star"></i>Аккаунт с подключенным пакетом
+          "Профессиональный" Создан 26 июля 2006 (#10761149) обновлялся 24 марта
+          2018.
+          <a
+            href="https://static8.depositphotos.com/1472772/996/i/600/depositphotos_9968262-stock-photo-colorful-gift-boxes-over-white.jpg"
+            target="_blanck"
+            >Подарить пакет "Профессиональный"</a
+          >
+        </h4>
+      </span>
+      <br />
+      <hr />
+    </div>
+    <div id="info">
+      <div id="block1">
+        <span class="block1">имя:</span>
+        <span class="block1">дата рождения:</span>
+        <span class="block1">место нахождения:</span>
+        <span class="block1">сайт:</span>
+        <span class="block1">способы связи:</span>
+        <br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><span
+          class="block1"
+          >учебные заведения:</span
+        >
+      </div>
+      <div id="block2">
+        <span class="block2">Ilya Varlamov</span>
+        <span class="block2">7 января 1984</span>
+        <span class="block2"
+          ><a
```

Comment:
Incorrect code style
```suggestion
          <a
```